core-bundle/src/Resources/config/listener.yml

```diff
@@ -196,6 +196,14 @@ services:
         tags:
             - kernel.event_listener
 
+    contao.listener.preview_url_create:
```

Comment:
please use the class name as service name for any new services

Reply:
I have adjusted the defintion.